modules/overrides/userconfigurable/api/http.go

```diff
@@ -159,6 +181,10 @@ func writeError(w http.ResponseWriter, err error) {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
+	if ok := errors.Is(err, errConflictingRuntimeOverrides); ok {
```

Comment:
Does this allocate `ok` or is the compiler smart enough to skip it?

Reply:
Huh, I'm not entirely sure why I created a temporary variable here 😅 I probably used a different function before that returned 2 values instead of juts a boolean 🤷🏻 

Again nice catch! Surprised the linter didn't pick this up